ui/src/components/DashboardServices.js

```diff
@@ -0,0 +1,295 @@
+  padding: ${spacing.sp4};
+`;
+
+const ClickableServiceItemElement = styled(ServiceItemElement)`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  :hover {
+    background-color: ${(props) => props.theme.highlight};
+  }
+`;
+
+const NonHealthyServiceItemElement = styled.div`
+  cursor: pointer;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+
+  a {
+    text-decoration: none;
+    color: inherit;
+    width: 100%;
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const NonHealthyPopUp = styled.div`
+  display: flex;
+  flex-direction: column;
+  font-size: ${fontSize.base};
+
+  label {
+    width: 25%;
+    margin-right: ${spacing.sp8};
+    color: ${(props) => props.theme.textSecondary};
+    text-align: right;
+  }
+`;
+
+const NonHealthyPopUpTitle = styled.div`
+  font-weight: ${fontWeight.bold}
+  text-align: center;
+`;
+
+const NonHealthyPopUpItem = styled.div`
+  width: 100%;
+  display: flex;
+  margin: ${spacing.sp4} ${spacing.sp14};
+  align-items: center;
+`;
+
+const ClickableIcon = styled.i`
+  self-align: flex-end;
+`;
+
+const ServiceItem = ({
+  label,
+  status,
+  alerts,
+}: {
+  label: string,
+  status: Status,
+  alerts: Alert[],
+}) => {
+  let date = null;
+  const intl = useIntl();
+
+  const getStartDate = (alerts: Alert[]) => {
+    // Check we only have alerts and set startsAt properties
+    const filtered = alerts.filter((item) => item && item.startsAt);
+    // Sort to make sure we have oldest alert first
+    filtered.sort((a, b) => new Date(a.startsAt) - new Date(b.startsAt));
+
+    if (filtered[0] && filtered[0].startsAt) return filtered[0].startsAt;
+    return '';
+  };
+
+  if (alerts.length) date = getStartDate(alerts);
+
+  if (!alerts.length && status === STATUS_HEALTH)
+    return (
+      <ServiceItemElement aria-label={label}>
+        <ServiceItemLabelWrapper>
+          <CircleStatus status={status} />
+          <ServiceItemLabel>{label}</ServiceItemLabel>
+        </ServiceItemLabelWrapper>
+      </ServiceItemElement>
+    );
+  else
+    return (
+      <NonHealthyServiceItemElement>
+        <Tooltip
+          // Right placement to avoid Z index issues between left sidebar and tooltip or out of screen tooltip displays
+          placement="top"
+          overlayStyle={{
+            // em sizing to handle font-size change on large screens
+            width: '20em',
+            height: '100%',
+          }}
+          overlay={
+            <NonHealthyPopUp>
+              <NonHealthyPopUpTitle>
+                {intl.formatMessage({ id: 'view_details' })}
+              </NonHealthyPopUpTitle>
+              <NonHealthyPopUpItem>
+                <label>{intl.formatMessage({ id: 'severity' })}</label>
+                <StatusText status={status}>{status}</StatusText>
+              </NonHealthyPopUpItem>
+              {
+                // Checking date validity
+                date && (
```

Comment:
```suggestion
                alert[0].startsAt && (
```
if you want to make sure we have a startsAt property, we can check here.